src/strategies/degenzoo-erc721-all-tokens/index.ts

```diff
@@ -0,0 +1,50 @@
+import fetch from 'cross-fetch';
+import { getAddress } from '@ethersproject/address';
+
+export const author = 'aorfevre';
+export const version = '0.1';
+
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+
+  // initialize scores
+  const scores = {};
+  for (const address of addresses) {
+    scores[getAddress(address)] = 0;
+  }
+
+
+  async function getWalletScore(wallet) {
+    const url = `https://api.daomaker.com/get-dzoo-assets-by-wallet?wallet=${wallet}`;
```

Comment:
Hi @aorfevre This is not your API? maybe you could use `api` or `api-v2` strategy instead?

Reply:
This is my api.

Reply:
However, I did not use your api-v2 because it is targeting a single network.

DZOO is a cross chain project and the Voting power shall reflect that. 
This api : https://api.daomaker.com/get-dzoo-assets-by-wallet?wallet=${wallet} is checking assets of the user on all chains.

Do you still want me to use one of your other templates ?

Reply:
So right now, it is sending one request per address won't scale, can send all in one request? so instead can use `api-v2` maybe just `network` param in your code? which should handle all networks right?

Also make sure your API consider `snapshot` block number to return voting power, else voting power can be unpredictable and won't be able to replay the voting power in future